packages/react-cookie/src/useCookies.tsx

```diff
@@ -2,10 +2,10 @@ import { useContext, useEffect, useState, useRef, useMemo } from 'react';
 import { Cookie, CookieSetOptions } from 'universal-cookie';
 import CookiesContext from './CookiesContext';
 
-export default function useCookies(
+export default function useCookies<T = { [name: string]: any }>(
```

Comment:
We can use `Record` here to generate this object by mapping the index and value types

```suggestion
export default function useCookies<T = Record<string, any>>(
```